lib/galaxy/tool_util/verify/asserts/size.py

```diff
@@ -0,0 +1,9 @@
+import io
+import sys
+
+
+def assert_has_size(output_bytes, value, delta=0):
+    """Asserts the specified output has a size of the specified value"""
+    output_temp = io.BytesIO(output_bytes)
```

Comment:
`output_bytes` is supposed to be bytes already, is that not the case ? iow can you skip io.BytesIO ?

Reply:
also technically `sys.getsizeof` returns the memory used by variable, that's not the same as the file size

Reply:
I think len(output_bytes) should be accurate here.